__tests__/app.test.js

```diff
@@ -84,7 +84,52 @@ describe("GET /api", () => {
       .get("/not-a-route")
       .expect(404)
       .then((response) => {
-        expect(response.body.msg).toBe("path not found");
+        expect(response.body.msg).toBe("not found");
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+    test('returns an object', () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article
+          expect(article.constructor).toBe(Object)
+        });
+    })
+    test('returns an object with correct properties ', () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article
+            expect(article.hasOwnProperty('author')).toBe(true)
+            expect(article.hasOwnProperty('title')).toBe(true)
+            expect(article.hasOwnProperty('article_id')).toBe(true)
```

Comment:
It would be better to be more specific here and expect the article_id to be '1' - how do we know that the correct one is coming back otherwise?